src/content/guides/efk-stack-app-oidc/index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Configure oidc for efk-stack-app
+title: Configure OIDC for EFK Stack App
 description: Configure authentication with kibana and the efk-stack-app through OIDC
 date: 2020-10-30
 type: page
 weight: 50
 tags: ["recipe"]
 ---
-# Setup OIDC to authenticate with kibana and efk-stack-app
+# Setup OIDC to authenticate with Kibana and EFK Stack App
```

Comment:
```suggestion
# Set up OIDC to authenticate with Kibana and EFK Stack App
```

Not sure, but it's two words, no? I know, this is not in scope of the PR, but once we are here...